.editorconfig

```diff
@@ -1,2 +1,2 @@
 [*.{kt,kts}]
```

Comment:
Could you `git rebase` your branch with master? Then we won't have a merge commit and the history looks a little better.

Reply:
Done